controller/src/beerocks/bml/bml_defs.h

```diff
@@ -549,6 +553,45 @@ struct BML_STATS_ITER {
     struct BML_STATS *(*get_node)();
 };
 
+struct BML_DCS_NEIGHBOR_AP {
+    //char  ap_Radio[64];
+    //The Radio that detected the neighboring WiFi SSID.
+    char ap_SSID[64];
```

Comment:
Please use static const definitions instead of hard coded numbers

Reply:
This should also be changed now cause I doubt that @itayx already changed it in his branches.

Reply:
Actually I did

Reply:
So OK :)